### PR TITLE
TinyMCE: Add media_buttons parameter

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/so-widgets-bundle/issues/903

### CODE REVIEW

**Comment by AlexGStapleton** [comment]
To test this open so-widgets-bundle/widgets/editor/editor.php and replace:

'title' => array(
	'type' => 'text',
	'label' => __('Title', 'so-widgets-bundle'),
),

With:

```
'title' => array(
	'type' => 'text',
	'label' => __('Title', 'so-widgets-bundle'),
	'media_buttons' => false,
),
```

**Comment by AlexGStapleton** [comment]
Sorry, there was a typo in the previously provided test code. Please switch to the updated code.

**Review by Misplon — APPROVED** [review]

File: base/inc/fields/tinymce.class.php [hunk 1]
```diff
@@ -415,15 +425,18 @@ protected function render_field( $value, $instance ) {
...
+		}
+
 		$settings['baseURL'] = includes_url( 'js/tinymce' );
 		$settings['suffix'] = SCRIPT_DEBUG ? '' : '.min';
 		
 		?><div class="siteorigin-widget-tinymce-container"
-			<?php if ( ! empty( $media_buttons ) ) : ?>
-			   data-media-buttons="<?php echo esc_attr( json_encode( array( 'html' => $media_buttons ) ) ) ?>"
+			<?php if ( $this->media_buttons && ! empty( $media_buttons_html ) ) : ?>
+		test="123"
```
Comment (gregpriday) [inline base/inc/fields/tinymce.class.php:438]: I assume this test code shouldn't be here?

**Review by gregpriday — CHANGES_REQUESTED** [review]
Looks like there's some test code.

**Review by gregpriday — APPROVED** [review]

**Pull request merged**